src/issue/issue.reducer.js

```diff
@@ -165,6 +168,23 @@ export const issueReducer = (state = initialState, action = {}) => {
         error: action.payload,
         isPendingIssue: false,
       };
+    case SUBMIT_NEW_ISSUE.PENDING:
+      return {
+        ...state,
+        isPendingSubmitting: true,
+      };
+    case SUBMIT_NEW_ISSUE.SUCCESS:
+      return {
+        ...state,
+        isSubmitted: action.payload,
```

Comment:
This is so awesome. The only thing I can think of is I don't think we need to store `isSubmitted` into our redux store. I think the `SUCCESS` handler can just set `isSubmitted` back to false but at the same time also assign the `issue` parameter to our new issue object (creating an issue [returns the new issue object](https://developer.github.com/v3/issues/#response-3) as the response)